src/BloomBrowserUI/pageChooser/page-chooser-main.htm

```diff
@@ -34,6 +35,10 @@
       <div id="singlePagePreview"><img src="" id="preview">
         <div id="previewCaption" class="previewCaption"></div>
         <div id="previewDescriptionText" class="DescriptionText">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed vel sodales justo.</div>
+        <div id="convertLosesMaterial">Converting to this layout will cause some content to be lost.</div>
```

Comment:
John's mock-up has "text" in place of "some content" but the code is also detecting loss of pictures